UPGRADE-2.0.md

```diff
@@ -1,5 +1,38 @@
 # Upgrade from 1.10 to 2.0
 
+Version `2.x` is a complete rewrite of the existing code base. The public API has been trimmed down to a minimum.
+The preferred way of using the SDK is through our static API.
+
+Here is a simple example to get started:
+
+```php
+\Sentry\init(['dsn' => '___PUBLIC_DSN___' ]);
+
+try {
+    thisFunctionThrows(); // -> throw new \Exception('foo bar');
+} catch (\Exception $exception) {
+    \Sentry\captureException($exception);
+}
+
+\Sentry\configureScope(function (\Sentry\State\Scope $scope): void {
+     $scope->setTag('page_locale', 'de-at');
+     $scope->setUser(['email' => 'john.doe@example.com']);
+     $scope->setLevel(\Sentry\Severity::warning());
+     $scope->setExtra('character_name', 'Mighty Fighter');
+});
+
+\Sentry\addBreadcrumb(new Breadcrumb(Breadcrumb::LEVEL_ERROR, Breadcrumb::TYPE_ERROR, 'error_reporting', 'Message'));
+```
+
+The call to `\Sentry\init()` sets up global exception/error handlers and uncaught errors will be sent to Sentry.
+
+Version `>= 2.0` conforms the [Unified SDK API](https://docs.sentry.io/development/sdk-dev/unified-api/).
+It has fundamentally different concepts, it's no longer recommended to just use a `Client` unless you really know what you are doing.
+
+If you are still want to use Dependecy Injection you should inject and use `HubInterface` which comes closest to `Raven_Client` you knew before.
```

Comment:
I would remove this phrase entirely since it provides no real value to the user

Reply:
Well it would point them in the right direction to get started easily without having to look through the whole codebase what to use now.

Reply:
But the sentence is misleading since it's not true that `HubInterface` comes closest to the old `Raven_Client` (we have the `Client` class now). Also dependency injection is a topic that is out of scope for this library since it's something users have to implement themselves, so talking about it is useless.